examples/shardingsphere-example-generator/src/main/resources/config.yaml

```diff
@@ -16,7 +16,7 @@
 #
 
 output:
-  - ./examples/shardingsphere-example-generator/target/generated-sources/
+  - ./shardingsphere-example-generator/target/generated-sources/
```

Comment:
This example generation is part of the examples section, so it makes sense to generate examples in the Examples directory.